.github/workflows/tests.yml

```diff
@@ -3,22 +3,20 @@ on: [push, pull_request]
 jobs:
   build:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.8, 3.7, 3.6]
+    name: Python ${{ matrix.python-version }}
     steps:
-      - uses: actions/checkout@v1
-      - name: Install pyenv
-        uses: gabrielfalcao/pyenv-action@v5
-        with:
-          default: 3.8.2
-          versions: 3.8.2 3.7.7 3.6.10
-      - name: Install dependencies
-        uses: abatilo/actions-poetry@v1.5.0
-        with:
-          python_version: 3.8
-          poetry_version: 1.0.5
-          args: install
-      - name: Run tests
-        uses: abatilo/actions-poetry@v1.5.0
-        with:
-          python_version: 3.8
-          poetry_version: 1.0.5
-          args: run tox
+    - uses: actions/checkout@v2
+    - uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install Poetry
+      run: pip install poetry
+    - name: Install dependencies
+      run: |
+        poetry install
+        poetry add tox-gh-actions
```

Comment:
This will **potentially** (although in rare cases) change `poetry.lock` and the packages installed in the _virtualenv_ managed by Poetry. Maybe `poetry run pip install tox-gh-actions` will be a more careful way to install it, what do you think?

```suggestion
        poetry run pip install tox-gh-actions
```

Reply:
An alternative is to actually add __locally__ this package to `pyproject.toml` with `poetry add -D tox-gh-actions` and commit both the `pyptoject.toml` and `poetry.lock` (removing this line 29 completely).

Reply:
What do you think is better since **tox-gh-actions** is only needed in CI env.

Reply:
The first option: just `pip` installing it.